src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Commands used to run other germline variant callers on the illumina platinum
+# subset test data.
+
+# Change this to point to your GATK jar.
+GATK_JAR=~/oss/gatk-binary/GenomeAnalysisTK.jar
+
+# Unified genotyper
+time java -jar $GATK_JAR \
+    -T UnifiedGenotyper \
+    -R src/test/resources/illumina-platinum-na12878-extract/chr1.prefix.fa \
+    -I src/test/resources/illumina-platinum-na12878-extract/NA12878.10k_variants.plus_chr1_3M-3.1M.chr_fixed.bam \
+    -L chr1:1-6700000 \
+    -glm INDEL \
```

Comment:
I think there's some other issues too with paths, I'll fix this up

On Wed, Mar 2, 2016 at 11:34 AM, Arun Ahuja <notifications@github.com>
wrote:

> In
> src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh
> <https://github.com/hammerlab/guacamole/pull/392#discussion_r54749683>:
>
> > @@ -0,0 +1,35 @@
> > +#!/bin/bash
> > +
> > +# Commands used to run other germline variant callers on the illumina platinum
> > +# subset test data.
> > +
> > +# Change this to point to your GATK jar.
> > +GATK_JAR=~/oss/gatk-binary/GenomeAnalysisTK.jar
> > +
> > +# Unified genotyper
> > +time java -jar $GATK_JAR \
> > +    -T UnifiedGenotyper \
> > +    -R src/test/resources/illumina-platinum-na12878-extract/chr1.prefix.fa \
> > +    -I src/test/resources/illumina-platinum-na12878-extract/NA12878.10k_variants.plus_chr1_3M-3.1M.chr_fixed.bam \
> > +    -L chr1:1-6700000 \
> > +    -glm INDEL \
>
> Should this be 'BOTH'? Is this why UnifiedGenotyper had ~8 % sensitivity
> in our test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/guacamole/pull/392/files#r54749683>.
>